src/Evolution/Systems/RelativisticEuler/Valencia/Characteristics.hpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace RelativisticEuler {
+namespace Valencia {
+
+/*!
+ * \brief Compute the characteristic speeds for the Valencia formulation
+ * of the relativistic Euler system.
+ *
+ * The principal symbol of the system is diagonalized so that the elements of
```

Comment:
Is this "principal symbol" the dF/du matrix, also known as the flux jacobian? Maybe I've been reading the wrong literature, but I feel the phrase "principal symbol" is a bit obscure.

Reply:
For a general system \partial_t u + A^j \partial_j u = s(u), A^i are the characteristics matrices (aka Jordan blocks). Then, given a normal n_i, one defines the principal symbol P(n) := n_i A^i. 

For conservative systems, the characteristic matrices are the matrices A^j = \partial F^j/\partial u, and the principal symbol is constructed from the corresponding contraction.